examples/http-rust-outbound-http/outbound-http/src/lib.rs

```diff
@@ -1,17 +1,17 @@
 use anyhow::Result;
 use spin_sdk::{
-    http::{Request, Response},
+    http::{IntoResponse, Request},
     http_component,
 };
 
 /// Send an HTTP request and return the response.
 #[http_component]
-fn send_outbound(_req: Request) -> Result<Response> {
-    let mut res = spin_sdk::outbound_http::send_request(
+fn send_outbound(_req: Request) -> Result<impl IntoResponse> {
```

Comment:
running into `destination-not-allowed (error 1)` triggering this component.

Reply:
I don't get why that would be happening given the changes are unrelated but looking into it.

Reply:
Good catch. Here's the fix: https://github.com/fermyon/spin/pull/1887/commits/e7d8a175d9a5cd2233c46ab0a66da9829ef14111